src/Check/OpCacheMemory.php

```diff
@@ -19,10 +20,8 @@ class OpCacheMemory extends AbstractMemoryCheck
 {
     /**
      * OpCache information
-     *
-     * @var array
      */
-    private $opCacheInfo;
+    private array|bool $opCacheInfo = false;
```

Comment:
Perhaps we can change approach here: require this property to be a well-formed `array`, and populating it only when the check succeeded? 

Allowing `false` here seems wrong.

Reply:
that won't work, as it assigned to `opcache_get_status()` in `check()` method, which can be `false`, mark only `array` will cause Fatal error, see https://3v4l.org/I6hHI

Reply:
OK, let's roll with the current implementation, for now :)